src/Searcher.tsx

```diff
@@ -133,8 +133,10 @@ export class Searcher {
     }
 
     clearSearch() {
-        let searchInput: any = document.getElementById("SearchInputText");
-        searchInput.value = "";
-        this.clearSearchFunc(searchInput.value);
+        let searchInput: any = document.getElementsByClassName("SearchInputText");
```

Comment:
Let's use `ref` instead, see [this document](https://facebook.github.io/react/docs/refs-and-the-dom.html) for details. Maybe name it as `this.searchInputField`.